doc/string/chop.rdoc

```diff
@@ -0,0 +1,16 @@
+Returns a new string copied from +self+, with trailing characters possibly removed.
+
+Removes <tt>"\r\n"</tt> if those are the last two characters.
+
+  "abc\r\n".chop      # => "abc"
+  "тест\r\n".chop     # => "тест"
+  "こんにちは\r\n".chop # => "こんにちは"
+
+Otherwise removes the last character if it exists.
+
+  'abcd'.chop     # => "abc"
+  'тест'.chop     # => "тес"
+  'こんにちは'.chop # => "こんにち"
+  ''.chop         # => ""
+
+String#chomp is sometimes a safer choice.
```

Comment:
I think you need to elaborate on this, maybe something like:

```suggestion
If you only need to remove the newline separator at the end of the string, String#chomp is a better alternative.
```

Reply:
Fixed.